g2p/mappings/langs/font-encodings/fn_unicode.csv

```diff
@@ -1,21 +1,40 @@
+\uF000,x\u030c
 \uF101,c\u0313
-\uF102,č\u0313
+\uF102,c\u030c\u0313
 \uF103,g\u0313
+\uF104,j\u030c\u0313
 \uF105,k\u0313
 \uF106,l\u0315
 \uF107,m\u0313
 \uF108,n\u0313
 \uF109,ŋ\u0313
-\uF110,z\u0313
-\uF111,\u019b\u0313
 \uF10A,p\u0313
 \uF10B,q\u0313
 \uF10C,r\u0313
 \uF10D,t\u0315
 \uF10E,w\u0313
 \uF10F,y\u0313
-\uF132,\u1DBF
+\uF110,z\u0313
+\uF111,ƛ\u0313
+\uF112,ɣ\u0313
+\uF113,ʕ\u0315
```

Comment:
I'm unsure about this one. Here is the glyph from the font:

![Screengrab of pharyngeal_glottalized from FNuni_2.3](https://user-images.githubusercontent.com/2294397/199347512-06413b62-c401-439b-8575-7806a98b8e59.png)

All other characters in this series combine a base letter with [◌̓ U+0313  COMBINING COMMA ABOVE](https://www.compart.com/en/unicode/U+0313), however I have opted for [◌̕ U+0315  COMBINING COMMA ABOVE RIGHT](https://www.compart.com/en/unicode/U+0313) for this one.

Compare:

| Combing character | Result |
|-|-|
| U+0313 | ʕ̓ |
| U+0315 | ʕ̕ |

Reply:
Based on inspecting the image at https://fnel.arts.ubc.ca/resources/font/ I have to agree with your choice. This is the character you get with Shift+Option 9, right? It's an odd inconsistency, but if it's what's been agreed on we should not deviate.

Reply:
Yea I feel like for line height reasons some of these tall characters opted for putting the comma above right. This sounds reasonable. `t` also does it strangely though